docs/custom-telemetry.md

```diff
@@ -0,0 +1,43 @@
+## Custom Telemetry With Scarf’s HTTP API<a id="custom-telemetry-with-scarfs-http-api"></a>
+
+Scarf provides you the ability to collect custom telemetry from within your application by utilizing our API. To enable this you'll need a Scarf account and a File Package created. Enable Event Collection Only. 
+
+Once this has been done, you are able to send telemetry data and associate it with the File Package you just created. Scarf has published and maintains an example in [bash or shell](https://github.com/scarf-sh/scarf-shell/), included below is an example Python function developed by one of our customers.
+
+_\*\*Thank you to Unstructured for their wonderful documentation and code samples. We have borrowed some of that as an example here. All rights reserved to Unstructured._
```

Comment:
```suggestion
```

Just removing this line. It's actually not "all rights reserved", as the code is open source.